ignite-base/package.json.template

```diff
@@ -62,7 +62,8 @@
     "reactotron-redux": "^1.5.2",
     "reactotron-redux-saga": "^1.5.2",
     "snazzy": "^5.0.0",
-    "standard-flow": "^1.0.0"
+    "standard-flow": "^1.0.0",
```

Comment:
Ignite default app gave me:

```
$ git commit -m "Re-added swipe cards git url as a dependency"

> giftwishlist@0.0.1 lint /Users/mike/Code/exampleapp
> standard --verbose | snazzy

sh: standard: command not found
```
I couldn't commit anything to Git until I installed 'standard'.